overlord/overlord.go

```diff
@@ -401,9 +403,24 @@ func (o *Overlord) Loop() {
 				return nil
 			case <-o.ensureTimer.C:
 			case <-o.pruneTicker.C:
+				if preseed {
+					// in preseed mode avoid setting StartOfOperationTime (it's
+					// an error), and don't Prune.
+					continue
+				}
 				st := o.State()
 				st.Lock()
-				st.Prune(pruneWait, abortWait, pruneMaxChanges)
+				var operationalTime time.Time
+				// account for deviceMgr == nil as it's not always present in
+				// the tests.
+				if o.deviceMgr != nil {
+					var err error
+					operationalTime, err = o.deviceMgr.StartOfOperationTime()
```

Comment:
I think I would prefer if we called this once in Overlord.StartUp and stored the value on the overlord for reuse. It just feels strange to set it to a value that is +10 mins from when we actually started